src/test/util/mod.rs

```diff
@@ -82,14 +82,10 @@ impl TestClient {
             .await;
         session.mark_dirty();
 
-        let is_master =
-            RunCommand::new("admin".into(), doc! { "isMaster":  1 }, None, None).unwrap();
+        let hello = RunCommand::new("admin".into(), doc! { "isMaster":  1 }, None, None).unwrap();
```

Comment:
Should this be using `"hello"`?

Reply:
This one I thought was required, since it's essentially an initial handshake. I realized that I can just reuse the `hello_command` method though, so I've updated it to that.